src/client/instant-meilisearch-client.ts

```diff
@@ -43,6 +38,12 @@ export function instantMeiliSearch(
     clientAgents,
   })
 
+  const searchCache = SearchCache()
+  // create search resolver with included cache
+  const searchResolver = SearchResolver(meilisearchClient, searchCache)
+
+  let defaultFacetDistribution: any
```

Comment:
Can `any` in typescript be considered bad practice?

Reply:
It can but it's not rare to see it in some places where it is hard to let typescript infer the type. But I actually tried it again and compared to before it doesn't seem to scream anymore so I'll update it!